src/blockchain_db/blockchain_db.h

```diff
@@ -186,7 +191,8 @@ struct txpool_tx_meta_t
     return matches_category(get_relay_method(), category);
   }
 };
-
+static_assert(sizeof(txpool_tx_meta_t) == 192, "possible DB migration needed for changes to txpool_tx_meta_t");
+static_assert(offsetof(txpool_tx_meta_t, valid_input_verification_id) % 32 == 0, "verif ID wrong alignment");
```

Comment:
Why `% 32` when the comment above says `til 160 bytes`? Maybe just check that the offset is exactly 160 bytes?

Reply:
This is fair